component/capacity.libsonnet

```diff
@@ -56,6 +57,54 @@ local exprMap = {
 
   ClusterCpuUsageHigh: function(arg) '%s < %f * %s' % [ cpuIdle, arg.factor, arg.threshold ],
   ExpectClusterCpuUsageHigh: function(arg) '%s < %f * %s' % [ predict(cpuIdle, range=arg.range, predict=arg.predict), arg.factor, arg.threshold ],
+
+  ClusterHasUnusedNodes: function(arg)
+    |||
+      min(
+        (
+          label_replace(
+            (%s - %s) / %s
+          , "resource", "pods", "", "")
+        ) or (
+          label_replace(
+            (%s - %s) / %s
+          , "resource", "requested_memory", "", "")
+        ) or (
+          label_replace(
+            (%s - %s) / %s
+          , "resource", "requested_cpu", "", "")
+        ) or (
+          label_replace(
+            %s / %s
+          , "resource", "memory", "", "")
+        ) or (
+          label_replace(
+            %s / %s
+          , "resource", "cpu", "", "")
+        )
+      ) > %f
+    |||
+    % [
+      podCapacity,
+      podCount,
+      getExpr('PodCapacity', 'TooManyPods').threshold,
```

Comment:
IMO using the threshold from the other alerts is a bit too much magic. If someone decides he only wants to be notified if the cluster can only schedule 10 more pods, this alert will break (We will alert very early) and it will not be clear why.

Reply:
Generally I think that would be cleaner. But that would count as a breaking change so I don't know..  We should definitely first check if anyone currently overwrites them. If yes I don't think it's worth breaking this

Reply:
It's unused according to my gws checkout.

Reply:
I'll break it in a separate PR though if that's ok with you.

Reply:
Yes I agree. Let's remove them. But definitely in a separate PR